Description: higan/target-tomoko-GNUmakefile fixes
 * Allow installation of files under fakeroot.
 * Create $(prefix)/share/applications/ folder.
Author: Tobias Hansen <thansen@debian.org>

--- a/higan/target-tomoko/GNUmakefile
+++ b/higan/target-tomoko/GNUmakefile
@@ -84,9 +84,7 @@
 endif
 
 install:
-ifeq ($(shell id -un),root)
-	$(error "make install should not be run as root")
-else ifeq ($(platform),windows)
+ifeq ($(platform),windows)
 else ifeq ($(platform),macosx)
 	mkdir -p ~/Library/Application\ Support/$(name)/
 	mkdir -p ~/Emulation/System/
